cypress.config.ts

```diff
@@ -17,8 +17,11 @@ export default defineConfig({
   reporterOptions: {
     configFile: 'reporter-config.json',
   },
+  video: false,
   videoUploadOnPasses: false,
   taskTimeout: 60000,
+  viewportWidth: 1200,
+  viewportHeight: 850,
```

Comment:
some GDS components change shape on smaller screens and cypress’ default is really small